astroquery/utils/tap/conn/tapconn.py

```diff
@@ -654,12 +655,12 @@ def check_launch_response_status(self, response, debug,
         isError = False
         if response.status != expected_response_status:
             if debug:
-                print("ERROR: " + str(response.status) + ": " +
+                log.debug("ERROR: " + str(response.status) + ": " +

```

Comment:
this can stay, as it would only print when `debug=True` is used, which is presumably not the case during production usage